specification/dimensions/commitmentutilization.md

```diff
@@ -0,0 +1,37 @@
+# Commitment Utilization
+
+Commitment Utilization indicates whether a charge utilized a commitment during the charge period. Commitment Utilization applies to all commitments and commitment types applied to the current charge. This includes but is not limited to capacity commitments and commitment-based discounts represented by the Commitment Discount ID. Commitment Utilization is commonly used to identify wastage when a commitment has not been used and to calculate commitment utilization.
+
+The CommitmentUtilization column MUST be present and MUST NOT be null when a commitment is applied or CommitmentDiscountId is set. This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+CommitmentUtilization
+
+## Display Name
+
+Commitment Utilization
+
+## Description
+
+Indicates whether a charge utilized a commitment during the charge period.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
```

Comment:
@BenShy @flanakin Regarding Line 23

When an unused value is rolled over from one contract to a subsequent contract - is that an exception to the "MUST" criteria? I'm familiar with a pre-paid commitment with a 10% rollover clause; in practice - if the customer uses 92% of what they prepaid - the remaining 8% is added to the subsequent contract - but does NOT have to be purchased again.

This would get complicated in the scenario where a customer consumes 85% of what they pre-paid, rolls over 10% to a subsequent contract and has 5% that they have to add into their cost data.

Happy to work through scenarios if you believe it needs to be part of V1.0
```suggestion
| Column required | False |